app/libs/containerConfig.js

```diff
@@ -0,0 +1,145 @@
+// env constants
+import { execSync } from 'child_process';
+import os from 'os';
+import path from 'path';
+// import pump from 'pump';
+// import throughJSON from 'through-json';
+// import through from 'through2';
+
+// gigantum image name
+const imageLabel = 'gigantum/labmanager';
+const imageTag = process.env.IMAGE_TAG;
+
+let nvidiaConfig;
+
+if (os.platform() === 'linux') {
+  try {
+    const rawNvidiaOutput = execSync(
+      'nvidia-smi --query-gpu=driver_version --format=csv,noheader'
+    ).toString();
+    nvidiaConfig = `NVIDIA_DRIVER_VERSION=${rawNvidiaOutput.split('\n')[0]}`;
+  } catch (error) {
+    // TODO DC: Do we have a better standard way of reporting status?
+    // I think we should avoid bare catches
+    console.log('unable to run nvidia-smi - assuming no GPU');
+  }
+}
+
+const isWindows = os.platform() === 'win32';
+const hostDirectory = path.join(os.homedir(), 'gigantum');
+
+const envHost = isWindows
+  ? 'WINDOWS_HOST=1'
+  : `LOCAL_USER_ID=${os.userInfo().uid}`;
+
+const condaDir = 'CONDA_DIR=/opt/conda';
+const shell = 'SHELL=/bin/bash';
+const miniCondaVersion = 'MINICONDA_VERSION=4.3.31';
+const lc = 'LC_ALL=C.UTF-8';
+const lang = 'LANG=C.UTF-8';
+
+/**
+ * @param {object} dockerode
+ * @param {function} dockerode
```

Comment:
This is a typo, right?

```suggestion
 * @param {function} callback
```